src/util/make-knex.js

```diff
@@ -32,6 +34,24 @@ module.exports = function makeKnex(client) {
       return client.raw.apply(client, arguments)
     },
 
+    batchInsert: function(table, batch, chunkSize) {
+      chunkSize = _.isNumber(chunkSize) ? chunkSize : 1;
+
+      return this.transaction((tr) => {
+
+          //Avoid unnecessary call
+          if(chunkSize !== 1) {
+            batch = _.chunk(batch, chunkSize)
+          }
+
+          Promise.all(batch.map((items) => {
```

Comment:
returning promise from transaction should call trx.commit / trx.rollback automatically if I recall correctly